moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -288,11 +295,24 @@ bool PlanningPipeline::generatePlan(const planning_scene::PlanningSceneConstPtr&
                      req_adapter->getDescription().c_str(), status.message.c_str());
         break;
       }
+
+      // check for timeout
```

Comment:
Add a note here that it is assumed that the request adapters only consume a very small fraction of the allowed planning time and thus they don't get a budget assigned. The timeout check is mainly a sanity check